recipe/meta.yaml

```
@@ -87,7 +87,7 @@ outputs:
     requirements:
       host:
         # So that pin_compatible has something to work with:
-        - libgfortran-ng {{ ctng_gcc }}.*
+        - libgfortran{{ libgfortran_soname }} {{ ctng_gcc }}.*
```

For mutual exclusivity
```suggestion
        - libgfortran{{ libgfortran_soname }} {{ ctng_gcc }}.*
        - libgfortran-ng
```